R/mergeSEs.R

```diff
@@ -476,26 +476,26 @@ setMethod("mergeSEs", signature = c(x = "list"),
 
 # Input: tree data and TreeSE
 # Output: TreeSE
-.check_and_add_trees <- function(tse, trees_and_links, MARGIN, verbose){
+.check_and_add_trees <- function(tse, trees_and_links, by, verbose){
```

Comment:
This also, no need to change

Reply:
Check other functions also. No need to change internal functions if the name is handled before calling them